services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+    user = get_user_model().objects.create_user(username=username,
+                                                password=password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+    return user
+
+
+def get_user(user_id: int) -> User | str:
+    try:
+        return get_user_model().objects.get(pk=user_id)
+    except User.DoesNotExist:
+        return f"{user_id} does not exist"
```

Comment:
returning string here is not the best choice, with this implementation it may lead to errors in other places. So the best would be to exit the program, or just let exception happen and then try-catch it wherever you are calling this function